packages/microcosm/src/microcosm.js

```diff
@@ -154,6 +138,10 @@ class Microcosm extends Emitter implements Domain {
     }
   }
 
+  get state(): Object {
+    return this._recall(this.history.head)
+  }
```

Comment:
`repo.state` is now a getter. It'll return the latest snapshot for history. This means we never have to make sure parents and children are synced up.